src/Orchard.Web/Modules/Orchard.Azure.Authentication/Migrations/AzureActiveDirectorySyncPartMigrations.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Orchard.Data.Migration;
+using Orchard.Data.Migration.Schema;
+using Orchard.Environment.Extensions;
+using Orchard.Azure.Authentication.Models;
+using System;
+
+namespace Orchard.Azure.Authentication.Migrations
+{
+	public class AzureActiveDirectorySyncPartMigrations : DataMigrationImpl
+	{
+
+		public int Create()
+		{
+			SchemaBuilder.CreateTable(typeof(AzureActiveDirectorySyncPartRecord).Name,
```

Comment:
Hmm, this migration class is no longer used. It was supposed to have been removed from module anyway. So I will remove it if it still exists in the code.